packages/code-du-travail-data/dataset/README.md

```diff
@@ -40,5 +37,17 @@ $ node index.js > ../nomenclatures-`date +%Y%m%d`.json
 
 ```
 $ cd dataset/fiches_ministere_travail
+$ npm install
 $ node ministere-travail-extract-fiches.js > fiches-min-travail.json
 ```
+
+## Mettre à jour les fiches service-public
+
+- placer les fichiers "[professionnels](https://www.data.gouv.fr/fr/datasets/service-public-fr-guide-vos-droits-et-demarches-professionnels-entreprises/)" dans `./dataset/fiches_service_public/vosdroits-professionnels`
+- placer les fichiers "[particuliers](https://www.data.gouv.fr/fr/datasets/service-public-fr-guide-vos-droits-et-demarches-particuliers/)" dans `./dataset/fiches_service_public/vosdroits-particuliers`
+
+```
+$ cd dataset/fiches_service_public
+$ npm install
+$ node service-public-extract-fiches.js > fiches-sp-travail.json
+```
```

Comment:
This is locking like a monorepo BTW...
We can ask yarn to install all the `packages/code-du-travail-data/dataset/*` folders and run a build script command.

Reply:
il will not work for fiche-service-public, because we first need to download the xml source file before running the script

Reply:
> we first need to download the xml source file

Who/How do you download it ?